RELEASING.md

```diff
@@ -0,0 +1,40 @@
+- [Overview](#overview)
+- [Branching](#branching)
+  - [Release Branching](#release-branching)
+  - [Feature Branches](#feature-branches)
+- [Release Labels](#release-labels)
+- [Releasing](#releasing)
+
+## Overview
+
+This document explains the release strategy for artifacts in this organization.
+
+## Branching
+
+### Release Branching
+
+Given the current major release of 1.0, projects in this organization maintain the following active branches.
+
+* **main**: The next _major_ release. This is the branch where all merges take place and code moves fast.
+* **1.x**: The next _minor_ release. Once a change is merged into `main`, decide whether to backport it to `1.x`.
+* **1.0**: The _current_ release. In between minor releases, only hotfixes (e.g. security) are backported to `1.0`.
+
+Label PRs with the next major version label (e.g. `2.0.0`) and merge changes into `main`. Label PRs that you believe need to be backported as `1.x` and `1.0`. Backport PRs by checking out the versioned branch, cherry-pick changes and open a PR against each target backport branch.
```

Comment:
> Label PRs that you believe need to be backported as `1.x` and `1.0`

i think they should be `backport 1.x` or `backport 1.0` to try backport workflow first before doing it manually?

and label `2.0.0` and `v2.0.0` (under "Release Labels") both seem to be for the release of 2.0.0, what is the difference between them?

Reply:
Just copied this from https://github.com/opensearch-project/opensearch-js/blob/main/RELEASING.md Regarding label I believe they are same but yeah you are right should be consistent. Made the change.
Thanks!